# L7

Which enzyme eliminates two phosphates from the nucleotides for incorporation into DNA by DNA polymerase I?

phosphatase

Role of primase?

Initiates DNA chain

# L8

Leading strand - Continuous strand - Same direction as the fork movement.

Lagging strand - Discontinuous - Okazaki fragments - Opposite direction to the form movement.

Closing of the nick due to okazaki fragments:

1. RNA primer removed by 5'-3' exonuclease. (or RNAse H)
2. DNA polymerized by DNA polymerase I.
3. Nick sealed by ligase.

Helicase (DnaB)? Unwinding.

Primase (DnaG)? primer synthesis. 

Polymerase I? Filling gaps, remove primers.

Polymerase III? Strand elongation.

Ligase? Ligation. Joining together.

Gyrase (Topoisomerase II)? Supercoiling.

Cost of one helicase unwinding?

ATP $\rightarrow$ ADP + P~1~

Why DNA polymerase accurate?

1. Correct base pairs necessary.
2. RNA bases disallowed due to discriminator amino acids.

Anomalous base pairing arrangements?

Thymine (enol) $\rightarrow$ Guanine (keto)

Cytosine (imino) $\rightarrow$ Adenine (amino)

DNA polymerase proofreading mechanism?

DNA polymerase I: DNA polymerase active site and 3'-5' exonuclease active site.

Mispaired bases $\rightarrow$ Blocks further elongation $\rightarrow$ Polymerase slides back with the mispaired base in the exonuclease active site $\rightarrow$ Mispaired nucleotide is removed $\rightarrow$ polymerase activity is resumed.

Exonuclease? Cuts at the ends.

Endonuclease? Cuts at the insides.

# L9

Processivity?

Ability to catalyse consecutive reactions without releasing its substrate.

How many nucleotides per second for DNA polymerase?

1000

The speed of DNA synthesis is due to the processivity of DNA polymerase. It has two modes: putative (slow, one nucleotide at a time) and fast (processive, many).

Telomere ?

Protects chromosome ends from damage. Caps the ends of the chromosome. Unique DNA sequence repeated several times. 

End replication problem?

Okazaki fragments leave behind a small region that is unreplicated. (Due to RNA primers? or is the fixed length (is it) of okazaki fragments?)With each subsequent replication, information is lost. 

Telomerase? 

Composed of TERT: Telomerase reverse transcriptase, TER: telomerase RNA.

The RNA in telomerase is reverse transcribed in order to lengthen the telomeric DNA. 

After the telomerase RNA is reverse transcribed to DNA, it can act as a template synthesise the other strand. However, this too results in the formation of Okazaki fragments which then result in a 3' overhang.

Helicase loading?

G~1~ - Loading permitted, activation not permitted.

Rest (G~2~, S, M) - Loading inhibited, activation permitted.

Cell cycle eukaryotes?

G~1~ $\rightarrow$ S $\rightarrow$ G~2~ $\rightarrow$ M

